opentracing-api/src/field.rs

```diff
@@ -0,0 +1,119 @@
+use std::fmt;
+use std::error::Error;
+use std::str::FromStr;
+
+const FIELD_ERROR_KIND: &str = "error.kind";
+const FIELD_ERROR_OBJECT: &str = "error.object";
+const FIELD_EVENT: &str = "event";
+const FIELD_MESSAGE: &str = "message";
+const FIELD_STACK: &str = "stack";
+
+/// The following log fields are recommended for instrumentors who are trying to capture more
+/// information about a logged event. Tracers may expose additional features based on these
+/// standardized data points.
+#[derive(Debug, PartialEq)]
```

Comment:
Just a couple of days ago I found the [rust api guidelines](https://rust-lang-nursery.github.io/api-guidelines/about.html) which has a [section about derives](https://rust-lang-nursery.github.io/api-guidelines/interoperability.html#c-common-traits).

I think we want to follow those guidelines, what do you think?

Reply:
agreed, will add them!